plugins/scaffolder/api-report.md

```diff
@@ -271,31 +310,56 @@ export const repoPickerValidation: (
 // @public
 export const RepoUrlPickerFieldExtension: FieldExtensionComponent<
   string,
-  RepoUrlPickerUiOptions
+  {
+    allowedOwners?: string[] | undefined;
+    allowedOrganizations?: string[] | undefined;
+    allowedRepos?: string[] | undefined;
+    allowedHosts?: string[] | undefined;
+    requestUserCredentials?:
+      | {
+          additionalScopes?:
+            | {
+                azure?: string[] | undefined;
+                github?: string[] | undefined;
+                gitlab?: string[] | undefined;
+                bitbucket?: string[] | undefined;
+                gerrit?: string[] | undefined;
+              }
+            | undefined;
+          secretsKey: string;
+        }
+      | undefined;
+  }
 >;
 
 // @public
-export interface RepoUrlPickerUiOptions {
-  // (undocumented)
-  allowedHosts?: string[];
-  // (undocumented)
-  allowedOrganizations?: string[];
-  // (undocumented)
-  allowedOwners?: string[];
-  // (undocumented)
-  allowedRepos?: string[];
-  // (undocumented)
-  requestUserCredentials?: {
-    secretsKey: string;
-    additionalScopes?: {
-      gerrit?: string[];
-      github?: string[];
-      gitlab?: string[];
-      bitbucket?: string[];
-      azure?: string[];
-    };
+export type RepoUrlPickerUiOptions =
+  typeof RepoUrlPickerUiOptionsSchema.schemaType;
+
+// @public (undocumented)
+export const RepoUrlPickerUiOptionsSchema: {
+  jsonSchema: JSONSchema7;
```

Comment:
i'm still a little unsure about these names here. wonder if `schema` and `type` for fine. They're a little confusing I feel.